src/Request.php

```diff
@@ -281,8 +281,6 @@ public function loadXml($request)
             return false;
         }
 
-        // @see Laminas-12293 - disable external entities for security purposes
-        $loadEntities  = libxml_disable_entity_loader(true);
```

Comment:
This call should be done conditionally: on PHP 7.3, this still is required

Reply:
ah yeah of course! :-P

Reply:
i now run checks for this